r-package/R/dev_cria.R

```diff
@@ -0,0 +1,86 @@
+#' Transform Brazilian UF code to UF name
+#'
+#' @description
+#'
+#' Automatically change a column that contains brazillian UF codes (e.g. 11,31,41) 
+#' into a colunm that contains brazillian UF reduced names (e.g. AM, MG, PR). Soon: automatically changes a column 
+#' into the basedosdados aesthetics
+#' 
+#'
+#'
+#' @importFrom dplyr mutate case_when 
+#' @importFrom magrittr %>% 
+#' @import tibble
+#' @export
+#'
+#' @return No return.
+#'
+#' @param base a tibble that contains a column that should be changed from id_uf to sigla_uf. Dataframes and other data formats will trigger an error.
+#' @param coluna a tibble column that contains the actual values of id_uf to be converted in sigla_uf.
+#'
+#' @examples
+#'
+#' \dontrun{
+#' tibble%>%dev_cria_estados(id_uf)
+#'
+#' # you can also not use the pipe
+#' 
+#' dev_cria_estados()
+#'
+#' 
+#'
+#'
+#' }
+#'
+
+
+dev_cria <- function(base, coluna, oque=estado) {
+  
+  if(oque = estado){
```

Comment:
acho que esse teste não está sendo usado. O argumento também não está documentado, então se for continuar é preciso colocar lá.

Reply:
entao quando eu tava conversando com o dev da bd eu e ele discutimos dar maior abstracao para essa funcao. nesse caso, o parametro "oque" ia indicar como/o que a funcao deveria criar. ao invés de mudar a uf por exemplo, podia alterar uma variavel toda em caps para a primeira letra em caps e o resto em minusculo. A ideia era juntar tudo na mesma funcao ao invés de criar uma funcao para cada alteracao.

Reply:
acho que isso pode ser feito a nível de coluna, tipo `function(base, column)`, se vc quer aplicar pra o de para em  `sigla_uf` então `column='sigla_uf'`, era mais ou menos essa a ideia.